common.js

```diff
@@ -11,6 +11,7 @@ const submodules = [
   'events', // Events and emitter
   'flags', // Flags data type
   'fp', // Functional programming
+  'fs', // Recursive mkdir
```

Comment:
I think this file will include more file system related methods than just this implementation:
```suggestion
  'fs', // File System
```